Assets/Scripts/Dialogue scripts/Dialogue.cs

```diff
@@ -1,197 +1,184 @@
+                yield return new WaitForSeconds(0.0375f);
 
-            yield return new WaitForSeconds(0.0375f);
+                while (!Input.GetKeyDown(KeyCode.E))
+                {
+                    yield return null;
+                }
 
-            while (!Input.GetKeyDown(KeyCode.E))
-            {
                 yield return null;
             }
 
-            yield return null;
-        }
-
-        if (currentRoutine != null) StopCoroutine(currentRoutine);
+            if (currentRoutine != null) StopCoroutine(currentRoutine);
 
-        dialogueBox.SetActive(false);
+            _dialogueBox.SetActive(false);
 
-        if (_itemClass >= 1)
-        {
-            joInvScript.AddPage(gameObject);
+            if (_itemClass >= 1)
+            {
+                _joInvScript.AddPage(gameObject);
 
-            _myRenderer.enabled = false;
-        }
+                _myRenderer.enabled = false;
+            }
 
-        _myCollider.enabled = false;
+            _myCollider.enabled = false;
 
-        rayCast.SetActive(true);
+            _rayCast.SetActive(true);
 
-        if (_itemClass == 0)
-        {
-            _myCollider.enabled = true;
-            if (item != null) item.SetActive(true);
-            if (isIntro == true)
+            if (_itemClass == 0)
             {
-                _myRenderer.enabled = false;
-                _myCollider.enabled = false;
+                _myCollider.enabled = true;
+                if (_item != null) _item.SetActive(true);
+                if (isIntro == true)
+                {
+                    _myRenderer.enabled = false;
+                    _myCollider.enabled = false;
+                }
             }
-        }
-        if (_movement != null) _movement.enabled = true;
+            if (_movement != null) _movement.enabled = true;
         
-    }
+        }
 
-    public IEnumerator RunHiddenParagraphCycle()
-    {
-        dialogueBox.SetActive(true);
+        public IEnumerator RunHiddenParagraphCycle()
+        {
+            _dialogueBox.SetActive(true);
 
-        rayCast.SetActive(false);
+            _rayCast.SetActive(false);
 
-        int paragraphCounter = 0;
+            int paragraphCounter = 0;
 
-        Coroutine currentRoutine = null;
+            Coroutine currentRoutine = null;
 
-        while (paragraphCounter < _hiddenTextBox.Length)
-        {
-            if (currentRoutine != null) StopCoroutine(currentRoutine);
+            while (paragraphCounter < _hiddenTextBox.Length)
+            {
+                if (currentRoutine != null) StopCoroutine(currentRoutine);
 
-            currentRoutine = StartCoroutine(Play(_hiddenTextBox[paragraphCounter]));
+                currentRoutine = StartCoroutine(Play(_hiddenTextBox[paragraphCounter]));
 
-            ++paragraphCounter;
+                ++paragraphCounter;
 
-            yield return new WaitForSeconds(0.0375f);
+                yield return new WaitForSeconds(0.0375f);
```

Comment:
I assume this is the same magic number I mentioned previously - so see my previous comment on this.